src/components/Column.js

```diff
@@ -1,7 +1,19 @@
+import PropTypes from 'prop-types';
+
 import { ColumnDisplayName } from '../constants';
 
-export default function Column() {
+const Column = () => {
   return null;
-}
+};
+
+Column.propTypes = {
```

Comment:
We could also add `propTypes` and `defaultProps` for other props passed to column component as well to make it easier to read and understand. I'll pick this one up if you are focusing on other features.

Reply:
Yes, I think this is a good idea 'cause currently you can't rely on IDE's suggestions to insert a prop since they aren't listed in prop types.